torch/onnx/symbolic_opset9.py

```diff
@@ -2423,3 +2423,37 @@ def take(g, self, index):
     out = index_select(g, self_flattened, 0, index)
     out = reshape_as(g, out, index)
     return out
+
+
+def _kl_div_log_target(g, input, target):
+    diff_ = sub(g, target, input)
+    exp_ = exp(g, target)
+    output = mul(g, exp_, diff_)
+    return output
+
+
+def _kl_div_non_log_target(g, input, target):
+    log_ = log(g, target)
+    diff_ = sub(g, log_, input)
+    output_pos = mul(g, target, diff_)
+    zeros_ = zeros_like(g, output_pos)
+    mask_ = gt(g, target, g.op("Constant", value_t=torch.tensor(0)))
```

Comment:
Just curious - in the `gt` op we are checking against a `onnx::Constant` which contains 0. The type seems to be int. Do we need to do something to handle situations when `target` is a some other type, say float? onnx::Greater requires the two inputs to be of the same type.

Reply:
Yes, this is going to be handled by the scalar type analysis in PR: https://github.com/pytorch/pytorch/pull/37787

Reply:
Great.